README.md

```diff
@@ -172,6 +172,9 @@ bots. If you are a beginner developer, please use this boilerplate.
 
 - Easily update your code without having to restart the bot everytime.
 
+**Cool Looking Logs**
+![Log Image](https://i.imgur.com/LHu1DL7.png)
```

Comment:
Can you add this just before the header instead?

Reply:
I would just remove the cool looking logs and move this to the top of the readme like line 2